android/src/main/java/com/robinpowered/RNMDMManager/RNMobileDeviceManagerModule.java

```diff
@@ -62,6 +60,7 @@ public String getName() {
     }
 
     private boolean isMDMSupported() {
+        RestrictionsManager restrictionsManager = (RestrictionsManager) super.getReactApplicationContext().getSystemService(Context.RESTRICTIONS_SERVICE);
```

Comment:
No need for the `super` keyword unless it's overwritten by the class making the call to the method defined in both the child & super classes

Reply:
Only `getReactApplicationContext()`